game/songs.hh

```diff
@@ -77,11 +77,12 @@ class Songs {
 	size_t loadedSongs() const { return m_songs.size(); }
 
   private:
-  	void LoadCache();
+	void LoadCache();
 	void CacheSonglist();
 
 	class RestoreSel;
-	typedef std::vector<std::shared_ptr<Song> > SongVector;
+	using SongPtr = std::shared_ptr<Song>;
+	using SongVector = std::vector<SongPtr>;
```

Comment:
putting the container type in the name is usually a bad idea as you dons express the "what" but the "how". Songs? SongCollection?